### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added cart functionality
image
image

### CODE REVIEW

File: src/components/cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  return (
+    <div>
+      <div class="cart-container">
+        <div class="cart-items-container">
+          {cartState.cart.length>0?(
+            cartState.cart.map((cartItem) => {
+              const {_id,img,original_price,price,qty,title,deliveryCharge}=cartItem;
+              return (
+                <div class="cart-item card">
```
Comment (shobit1337) [inline src/components/cart/Cart.jsx:54]: Use key attribute here

File: src/components/cart/Cart.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  })
+
+  const cartHandler=(item,type)=>{
+    if(token){
+      updateCartHandler(token,item,type,cartDispatch)
+    }
+  }
+  
+
+  // useEffect(() => {
```
Comment (xsaurabhsuthar) [inline src/components/cart/Cart.jsx:33]: remove unused comments.

**Review by xsaurabhsuthar — APPROVED** [review]
* Good folder structure.
* Clean code.

**Review by shobit1337 — APPROVED** [review]
Looks good:
Some Suggestions:
- Removed Commented code before pushing to GitHub
- Remove all console.logs also, 
- Use Prettier for better visibility

**Pull request merged**